docs/src/components/pricing/PricingTable.tsx

```diff
@@ -873,40 +876,49 @@ export default function PricingTable({
               variant="outlined"
               fullWidth
               endIcon={<KeyboardArrowRightRounded />}
-              sx={{ py: 1, mt: 'auto' }}
+              sx={{ py: 1, mt: 'auto', mb:2 }}
             >
               Get started
             </Button>
           </Box>
           <ColumnHeadHighlight>
             <Recommended />
-            <PlanName plan="pro" />
-            <PlanPrice plan="pro" />
+            <Box >
+              <PlanName plan="pro" />
+              <PlanPrice plan="pro" />
+            </Box>
             <Button
               component={Link}
               noLinkStyle
               href="https://mui.com/store/items/material-ui-pro/"
               variant="contained"
               fullWidth
               endIcon={<KeyboardArrowRightRounded />}
-              sx={{ py: 1, mt: 'auto' }}
+              sx={{ py: 1, mt: 3 }}
             >
               Buy now
             </Button>
+            <Typography variant="body2" textAlign="center" color="text.secondary">
+              Flat price at 10 devs / application
+            </Typography>
```

Comment:
This feels out of place to me. Does it need to be mentioned at all here? If so, maybe it belongs up above the CTA button?